### PR TITLE
Cy/server scope

### PR DESCRIPTION
**Pull request by cy6erGn0m**

### CODE REVIEW

File: ktor-server/ktor-server-jetty/src/io/ktor/server/jetty/JettyKtorHandler.kt [hunk 1]
```diff
@@ -25,10 +25,9 @@ internal class JettyKtorHandler(
     private val dispatcher = DispatcherWithShutdown(executor.asCoroutineDispatcher())
     private val MULTI_PART_CONFIG = MultipartConfigElement(System.getProperty("java.io.tmpdir"))
```
Comment (e5l) [inline ktor-server/ktor-server-jetty/src/io/ktor/server/jetty/JettyKtorHandler.kt]: `MULTIPART_CONFIG`

File: ktor-server/ktor-server-host-common/src/io/ktor/server/engine/EngineContextCancellationHelper.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import kotlinx.coroutines.*
+import java.util.concurrent.*
+import kotlin.coroutines.*
+
+/**
+ * A helper class useful with an [ApplicationEngine] implementation to handle parent context cancellation
+ */
+@EngineAPI
+@KtorExperimentalAPI
+class EngineContextCancellationHelper(private val parent: CoroutineContext,
```
Comment (e5l) [inline ktor-server/ktor-server-host-common/src/io/ktor/server/engine/EngineContextCancellationHelper.kt]: code style

File: ktor-server/ktor-server-jetty/src/io/ktor/server/jetty/JettyApplicationEngine.kt [hunk 3]
```diff
@@ -8,7 +8,8 @@ import java.util.concurrent.*
 /**
  * [ApplicationEngine] implementation for running in a standalone Jetty
  */
-class JettyApplicationEngine(environment: ApplicationEngineEnvironment, configure: Configuration.() -> Unit) : JettyApplicationEngineBase(environment, configure) {
+class JettyApplicationEngine(environment: ApplicationEngineEnvironment, configure: Configuration.() -> Unit) :
```
Comment (e5l) [inline ktor-server/ktor-server-jetty/src/io/ktor/server/jetty/JettyApplicationEngine.kt]: code style

**Comment by cy6erGn0m** [comment]
Merged

**Pull request closed without merging**